angular-cli.json

```diff
@@ -20,9 +20,7 @@
       "styles": [
         "styles.css"
       ],
-      "scripts": [
-        "../node_modules/jquery/dist/jquery.min.js"
-      ],
+      "scripts": [],
```

Comment:
@Marauderer97 to reproduce the error: create a new branch and updated to master. Run `ng serve` or `ng test`, it will throw an error of multi-script loader. :)